src/uu/tail/Cargo.toml

```diff
@@ -18,12 +18,13 @@ path = "src/tail.rs"
 [dependencies]
 clap = { version = "3.2", features = ["wrap_help", "cargo"] }
 libc = "0.2.132"
-notify = { version = "=5.0.0-pre.16", features=["macos_kqueue"]}
-uucore = { version=">=0.0.15", package="uucore", path="../../uucore", features=["ringbuffer", "lines"] }
+memchr = "2.5.0"
+notify = { version = "=5.0.0-pre.16", features = ["macos_kqueue"] }
+uucore = { version = ">=0.0.15", package = "uucore", path = "../../uucore", features = ["ringbuffer", "lines"] }
```

Comment:
I'm all for fixing this spacing, but preferably in another PR and across the entire project at the same time, because we have some scripts that assume a certain spacing (for upgrading the version number of uucore for instance)

Reply:
fixed it. Should I create an issue for this?